### DISCUSSION TITLE
Newline at end of line

### DISCUSSION THREAD

**Discussion by svenssonjoel** [TheZoraiz/ascii-image-converter#23] [Ideas]
Hello, 

Is it possible to make the tool insert newline characters at the end of each line? And allow the user to choose how to terminate a line. 
(I would want \r\n, for example)

I see that it actually does insert newlines, sorry about that ;). But being able to choose additional characters or alternative ones for line termination would perhaps be good?

`ascii-image-converter -c image.file | awk '{printf "%s\r\n", $0}'` does the trick! 

Thanks!

**Comment by TheZoraiz** [comment]
Glad you were able to resolve the issue. As for the feature request, I don't think adding an entire new flag just for customized line termination would be appropriate. People who want to do it can easily do so with scripts like the one you've mentioned.

**Reply by svenssonjoel** [reply]
Yes! I think this is the best idea. I always forget that we can easily compose commands ;)  

Thanks for the tool. it's a lot of fun.
